atk/vendor.md

```diff
@@ -336,7 +336,7 @@ height to set for `self`
 # Returns
 
 `true` or `false` whether the size was set or not
-<!-- trait ComponentExt::fn connect_bounds_changed -->
+<!-- trait ComponentExt::fn bounds-changed -->
```

Comment:
IMHO it wrong

Reply:
What the hell! O.o

Reply:
All signal broke 😢

Reply:
Seems https://github.com/gtk-rs/gir/blob/master/src/codegen/doc/mod.rs#L295 need use `analysis.connect_name` if it possible

Reply:
Yep... Do you mind doing it? I guess I'll send the documentation PR once fixed and end the release meanwhile.

Reply:
Ok.

Reply:
Thanks!